content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -867,6 +873,50 @@ is disabled, `.spec.completions` is immutable.
 Use cases for elastic Indexed Jobs include batch workloads which require 
 scaling an indexed Job, such as MPI, Horovord, Ray, and PyTorch training jobs.
 
+### Pod Replacement Policy
+
+{{< feature-state for_k8s_version="v1.28" state="alpha" >}}
+
+{{< note >}}
+You can only set `podReplacementPolicy` on Jobs if you enable the `JobPodReplacementPolicy` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/).
+{{< /note >}}
+
+By default, the Job controller recreates pods as soon they are either failed or terminating (have a deletion timestamp).
+This means that, at a given time, the number of running Pods for the Jobs can be greater than `parallelism` or, if using Indexed Jobs, more than one running Pod per index, if some of the Pods are terminating.
+
+You may choose to create replacement pods only when the terminating pod is fully terminal (has `status.phase: Failed`). To do this, set the `.spec.podReplacementPolicy: Failed`.
+This will only recreate pods once they are terminated.  
+The default policy is `FailedOrTerminating`, meaning that the control plane creates replacement Pods upon deletion (`DeletionTimestamp != nil`).
+
+```yaml
+kind: Job
+metadata:
+  name: new
+  ...
+spec:
+  podReplacementPolicy: Failed
+  ...
+```
+
+You can inspect a new field in the JobStatus called `terminating`.  
+This will report the number pods that are currently terminating and is easily viewable in the status.
+
+```shell
+kubectl get jobs/myjob -o yaml
+```
+
+```yaml
+apiVersion: batch/v1
+kind: Job
+# .metadata and .spec omitted
+status:
+  terminating: 1 # if pod is terminating
+```
+
+When you use a [podFailurePolicy](#pod-failure-policy) in a Job, the Job will have a default `podReplacementPolicy` value of `Failed`, and this is the only policy allowed.
+If `JobPodReplacementPolicy` is disabled and `podFailurePolicy` is enabled, a Job will wait for terminating pods to be fully terminated before marking the pod as failed.
```

Comment:
```suggestion
When you use a [podFailurePolicy](#pod-failure-policy) in a Job, the Job has a default `podReplacementPolicy` value of `Failed`, and this is the only policy allowed.
If `JobPodReplacementPolicy` is disabled and `podFailurePolicy` is enabled, the Job waits for terminating pods to be fully terminated before marking the pod as failed.
```

Reply:
Yours and @sftim suggestions conflict here so maybe take a look over it and see if you are happy with the changes overall.

Reply:
You can omit my suggestion. We recommend not using future tense, but it's okay if the context needs the future tense 😄